docker-compose.yml

```diff
@@ -17,8 +17,32 @@ services:
       - frontend-dist:/usr/share/nginx/html
     extra_hosts:
       - "host.docker.internal:host-gateway"  # Linux compatibility
-
+  minio:
+    image: minio/minio
+    container_name: minio
+    ports:
+      - "9000:9000"   # S3 API
+      - "9001:9001"   # Web Console
+    environment:
+      MINIO_ROOT_USER: admin
```

Comment:
can we maybe extract such fields to .env file and generate it on the fly?

Reply:
Yes, I can do that. But I will push a new PR, all the Docker files need to be reorganized a bit.